CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-Update Docker security practices (Add HEALTHCHECK, Use Anonymous User, Use two-stage build)
\ No newline at end of file
+FIX: check ngsi version in configuration handler (#500)
```

Comment:
```suggestion
FIX: check ngsi version in configuration handler
```

As issue 500 has no meaning in this repository.

Reply:
fixed in https://github.com/telefonicaid/iotagent-ul/pull/445/commits/9eb342aac58f6b8409f5bcef34ce36c3b9748dee